flow/packet-headers/ipv4.yaml

```diff
@@ -403,3 +409,19 @@ components:
             default: 0
             features: [count, metric_tags]
           x-field-uid: 6
+    Flow.Ipv4.Auto:
+      description: |-
+        The OTG implementation can provide a system generated, value for this property.
+      type: object
+      required: [choice]
+      properties:
+        choice:
+          description: |- 
+            The method to be used to provide the system generated value.
+            
+            The dhcp option populates the field based on the dynamic IPv4 address that has been assigned to the DHCPv4 client by a DHCPv4 server.
+          type: string
+          x-enum:
+            dhcp:
+              x-field-uid: 1
+          x-field-uid: 1
```

Comment:
does this automatically default to dhcp?

Reply:
AFAICT the only choice here is dhcp, so why force the user to specify it? Why even have a choice?

Reply:
@lthompson-keysight For the Traffic use case:
In OTG model, we need to push the control-plane & data-plane config at the time of setconfig
For use case, when a user may to generate traffic as traffic Tx endpoint as a DHCP Client interface, which does have not an IP address yet. Once control-plane CP) is Up, its Data-plane module (DP<--> CP) queries to CP for the IP.
It will be good even, if possible, user does not need to explicitly mention AutoDhcp() however, 
internally traffic module finds out that its dhcp interface and it needs to get IP from CP module?